api/AuthenticatorAssertionResponse.json

```diff
@@ -16,7 +16,15 @@
             ]
           },
           "chrome_android": {
-            "version_added": null
+            "version_added": "65",
+            "notes": "Only support USB U2F tokens",
+            "flags": [
+              {
+                "type": "preference",
+                "value_to_set": "true",
```

Comment:
I thought Chrome prefs used `"Enabled"` and `"Disabled"` and that's what I found on desktop and Android, version 71. Is this changing?